ProjFS.Mac/Scripts/Build.sh

```diff
@@ -13,6 +18,9 @@ COVERAGEDIR=$ROOTDIR/BuildOutput/ProjFS.Mac/Coverage
 
 PROJFS=$SRCDIR/ProjFS.Mac
 
+echo "Generating PrjFSVersion.h as $VERSION..."
+$SCRIPTDIR/GeneratePrjFSVersionHeader.sh $VERSION || exit 1
```

Comment:
@jrbriggs FYI I made a change here so that `ProjFS.Mac/Scripts/Build.sh` will generate a header file that contains the VFS4G/PrjFS version.

I also updated `PrjFSKextLogDaemon` and the kext/user-mode to use this version string rather than a hardcoded version.

Let me know if you see any issues with this approach.  I will be filing a follow up issue to also generate an xcconfig file so that the correct version can be stamped in the plist file.

Reply:
@wilbaker This approach makes sense to me, thanks!